src/emailpro/account/update/emailpro-account-update.controller.js

```diff
@@ -1,5 +1,6 @@
 angular.module('Module.emailpro.controllers').controller('EmailProUpdateAccountCtrl', ($q, $scope, $stateParams, $translate, EmailPro, EmailProPassword) => {
   const originalValues = angular.copy($scope.currentActionData);
+  $scope.mxPlan = {};
```

Comment:
Is it possible to use `this` instead to prevent putting everything in the `$scope` ?

Reply:
I think it would require a biiiiiiig refactoring, but this section deserves one anyway